services/meta/config.go

```diff
@@ -19,19 +19,21 @@ const (
 type Config struct {
 	Dir string `toml:"dir"`
 
-	RetentionAutoCreate bool `toml:"retention-autocreate"`
-	LoggingEnabled      bool `toml:"logging-enabled"`
-	PprofEnabled        bool `toml:"pprof-enabled"`
+	RetentionAutoCreate        bool   `toml:"retention-autocreate"`
+	DefaultRetentionPolicyName string `toml:"default-retention-policy-name"`
+	LoggingEnabled             bool   `toml:"logging-enabled"`
+	PprofEnabled               bool   `toml:"pprof-enabled"`
 
 	LeaseDuration toml.Duration `toml:"lease-duration"`
 }
 
 // NewConfig builds a new configuration with default values.
 func NewConfig() *Config {
 	return &Config{
-		RetentionAutoCreate: true,
-		LeaseDuration:       toml.Duration(DefaultLeaseDuration),
-		LoggingEnabled:      DefaultLoggingEnabled,
+		RetentionAutoCreate:        true,
```

Comment:
The idiom with Influx configs is to add `DefaultDefaultRetentionPolicyName = "autogen"` as a constant (even though the name is silly), and then set `DefaultRetentionPolicyName: DefaultDefaultRetentionPolicyName` in the constructor. 

That way other packages can figure out the default without having to construct a config.